File: fib.erl
-module(fib).
-export([fib/1, main/0]).

fib(0) -> 0;
fib(1) -> 1;
fib(N) -> fib(N-1) + fib(N-2).

main() ->
    io:format("~p~n", [fib(10)]).

File: fib.zig
const std = @import("std");
fn fib(n: u64) u64 {
    if (n <= 1) return n;
    return fib(n - 1) + fib(n - 2);
}
pub fn main() void {
    const stdout = std.io.getStdOut().writer();
    stdout.print("{}\n", .{fib(10)}) catch {};
}
